app/services/idv/steps/threat_metrix_step_helper.rb

```diff
@@ -11,8 +11,8 @@ def threatmetrix_view_variables
         }
       end
 
-      def generate_threatmetrix_session_id
-        idv_session.threatmetrix_session_id = SecureRandom.uuid if !@ssn_form.updating_ssn?
+      def generate_threatmetrix_session_id(updating_ssn)
+        idv_session.threatmetrix_session_id = SecureRandom.uuid if !updating_ssn
```

Comment:
Instead of reaching into SsnController instance variables, pass in updating_ssn.